easypost/carrier_account.py

```diff
@@ -16,6 +17,19 @@
 )
 
 
+def _select_carrier_account_creation_endpoint(carrier_account_type: Optional[Any]) -> str:
+    """Determines which API endpoint to use for the creation call."""
+    carriers_with_custom_workflows = [
+        "FedExAccount",
+        "UpsAccount",
+    ]
```

Comment:
These should probably be constants in each lib where possible? Small performance improvements.

Reply:
Resolved